src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/Repositories/FeedbackAnalysisResultRepository.cs

```diff
@@ -24,16 +24,4 @@ public new IQueryable<FeedbackAnalysisResult> Get(Expression<Func<FeedbackAnalys
     {
         return base.GetByIdAsync(feedbackId, queryOptions, cancellationToken);
     }
-
-    public new ValueTask<FeedbackAnalysisResult> CreateAsync(FeedbackAnalysisResult feedbackAnalysisResult, CommandOptions commandOptions = default,
-        CancellationToken cancellationToken = default)
-    {
-        return base.CreateAsync(feedbackAnalysisResult, commandOptions, cancellationToken);
-    }
-
-    public new ValueTask<FeedbackAnalysisResult?> DeleteByIdAsync(Guid feedbackId, CommandOptions commandOptions = default,
-        CancellationToken cancellationToken = default)
-    {
-        return base.DeleteByIdAsync(feedbackId, commandOptions, cancellationToken);
-    }
```

Comment:
Why did you remove Create method ? 

Feedback analysis result won't be updated or deleted, but we should definitely be able to create one ))